src/main/routes/routes.ts

```diff
@@ -67,6 +67,7 @@ export default function(app: Application): void {
   app.get('/alphabetical-search', app.locals.container.cradle.alphabeticalSearchController.get);
   app.post('/alphabetical-search', app.locals.container.cradle.alphabeticalSearchController.post);
   // app.get('/case-event-glossary', app.locals.container.cradle.caseEventGlossaryController.get);
+  app.get('/case-standards-list', app.locals.container.cradle.primaryHealthListController.get);
```

Comment:
I think if you want to reuse the controller for primary health list, you should at least change the name of the controller so it is generic too

Reply:
Agreed. Renamed Controller and Service.